schema.md

```diff
@@ -0,0 +1,136 @@
+# Sparkle Motion schema (WIP)
+
+All record types have `createdAt` (and maybe `modifiedAt`?).
+
+### User
+
+```js
+{
+  "id": ___,
+  "username": "xian",
```

Comment:
If we're gonna go this far, we might as well have avatars as well. Photos we upload ahead of time, or generated image hashes kinda like slack/GitHub do?

Reply:
Totally. It's easy to upload a photo from the camera in html these days right?